modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -43,7 +43,7 @@ data:
       mirror-by-digest-only = true
 ----
 <1> The mirror registry’s certificate used when creating the mirror registry.
-<2> The configuration for the mirror registry.
+<2> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to `/etc/containers/registries.conf` in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when passed to the installer. The Assisted Service pod running on the HUB cluster fetches the container images from the configured mirror registry.  
```

Comment:
```suggestion
<2> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to `/etc/containers/registries.conf` in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when passed to the installation program. The Assisted Service pod running on the HUB cluster fetches the container images from the configured mirror registry.  
```

Reply:
Thanks. I'll run this by the SME tomorrow. It may be Assisted Installer.